azurerm/resource_arm_policy_set_definition_test.go

```diff
@@ -0,0 +1,122 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMPolicySetDefinition_basic(t *testing.T) {
```

Comment:
could we add a test for a built-in policy definition too? https://github.com/terraform-providers/terraform-provider-azurerm/pull/2535/files#diff-3a9bf2b3e79004afd0f088b0ccc7b695R49

Reply:
👍 
 I had actually referenced a built-in policy. I've now added two distinct acceptance tests. One building a Policy Definition Set with a custom policy, another referencing a built-in policy. Tests are named accordingly.